lib/playgroundbook_renderer/page_processor.rb

```diff
@@ -0,0 +1,15 @@
+module Playgroundbook
+  class PageProcessor
+    def process_page(page_contents)
+      # Three cases we need to look for:
+      # - Extraneous newlines before /*:
+      # - Extraneous newlines after */
+      # - Extraneous newlines either before or after //:
+      page_contents
```

Comment:
Maybe break this out into a `strip_extraneous_newlines` method to make it more explicit, since this main `process_page` method will likely grow to have more responsibilities eventually.